svelte/lib/src/components/rating/Rating.svelte

```diff
@@ -26,52 +27,21 @@
 	});
 
 	const {
-		stores: {
-			tabindex$,
-			maxRating$,
-			visibleRating$,
-			ariaValueText$,
-			readonly$,
-			disabled$,
-			interactive$,
-			stars$,
-			className$,
-			slotStar$,
-			ariaLabel$,
-			ariaLabelledBy$,
-		},
-		actions: {handleKey, leave, hover, click},
+		stores: {visibleRating$, stars$, slotStar$},
+		directives: {containerDirective, starDirective},
 		patchChangedProps,
 	} = widget;
 	$: patchChangedProps($$props);
+
+	const containerAttributes = ssrDirectiveAttributes(containerDirective);
 </script>
 
-<div
-	role="slider"
-	class="d-inline-flex au-rating {$className$}"
-	tabindex={$tabindex$}
-	aria-valuemin={0}
-	aria-valuemax={$maxRating$}
-	aria-valuenow={$visibleRating$}
-	aria-valuetext={$ariaValueText$}
-	aria-readonly={$readonly$ || undefined}
-	aria-disabled={$disabled$ || undefined}
-	aria-label={$ariaLabel$ || undefined}
-	aria-labelledby={$ariaLabelledBy$ || undefined}
-	on:keydown={handleKey}
-	on:mouseleave={leave}
->
+<div use:containerDirective {...containerAttributes} class="d-inline-flex">
```

Comment:
Having both `{...containerAttributes}` (which includes `class` in SSR) and `class="d-inline-flex` will not work well, I think.